python/apps/taiga/src/taiga/workflows/repositories.py

```diff
@@ -262,3 +262,17 @@ def get_status(
         return qs.get()
     except WorkflowStatus.DoesNotExist:
         return None
+
+
+##########################################################
+# WorkflowStatus - update workflow status
+##########################################################
+
+
+@sync_to_async
+def update_workflow_status(status: WorkflowStatus, values: dict[str, Any] = {}) -> WorkflowStatus:
```

Comment:
according to our conventions, we agreed to start writing async repositories with the new django-orm async api, instead of using the decorator (when possible).